jazz-build-module/service-configuration-data-loader.groovy

```diff
@@ -217,4 +217,31 @@ def setServiceName(serviceName){
 	service_name = serviceName
 }
 
+def setKinesisStream(config){
```

Comment:
This is a nice abstraction, looking good.

Reply:
Thanks Surya